dist/windows/InstallerTemplate.ps1

```diff
@@ -205,13 +205,13 @@ if ($Install)
 	New-Item -ItemType Directory -Path "$InstallPath"						-ErrorAction SilentlyContinue	| Out-Null
 
 	# writing ZIP file to disk
-	$TempFilePath =									[System.IO.Path]::GetTempFileName()
+	$TempFilePath =	[System.IO.Path]::GetTempFileName().TrimEnd("tmp") + "zip"
 	Write-Host "  Writing temporary ZIP file: $TempFilePath"
 	$CompressedFileContentAsBytes =	[System.Convert]::FromBase64String($CompressedFileContentInBase64)
 	[System.IO.File]::WriteAllBytes("$TempFilePath", $CompressedFileContentAsBytes)
 	
 	Write-Host "  Extracting ZIP file to: $InstallPath"
-	Expand-Archive "$TempFilePath" -OutputPath $InstallPath -Force -ShowProgress
+	Expand-Archive "$TempFilePath" -DestinationPath $InstallPath -Force
```

Comment:
Please use the FQN (Fully Qulified Name) as suggested here: https://github.com/ghdl/ghdl/issues/457#issuecomment-354464345

Otherwise this fix will break all PowerShell setups that have PSCX installed like mine :)

Reply:
If you are still online, you can reach me in Gitter.im for a direct discussion.